build.gradle.kts

```diff
@@ -8,12 +7,12 @@ plugins {
     `java-library`
     id("olf.build-logic")
     id("com.diffplug.spotless") version "6.18.0"
-    id("com.github.johnrengelman.shadow") version "8.1.1"
+    id("io.github.goooler.shadow") version "8.1.7"
     id("net.minecrell.plugin-yml.bukkit") version "0.5.3"
-    id("xyz.jpenilla.run-paper") version "2.1.0"
+    id("xyz.jpenilla.run-paper") version "2.3.0"
     idea
 
-    id("io.papermc.hangar-publish-plugin") version "0.0.5"
+    id("io.papermc.hangar-publish-plugin") version "0.1.2"
```

Comment:
Are the changes in relation to 1.20.6 support?
Or can they be made in advance ?

This would make the changes more isolated